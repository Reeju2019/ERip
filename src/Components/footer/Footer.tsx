import React from "react";
import "./footer.css";
import { footerData } from "./footerData";



const Footer = () => {
  return (
    <>
      <section className="footer">
        {footerData.map((e: any) => {
          return (
            <div
              className="container-fluid g-0 footer-container"
              style={{
                paddingTop: "50px",
                paddingBottom: "50px",
                display: "flex",
                backgroundColor: "#24252a",
              }}
            >
              <div className="first-footer ">
                <img
                  src={e.first_footer.logo}
                  alt="white-logo"
                  className="erip-logo"
                />
                <p className="footer-text">{e.footer_text}</p>
                <i className="fa-brands fa-twitter"></i>
                <i className="facebook fab fa-facebook-square"></i>
                <i className="Instagram fab fa-instagram"></i>
                <i className="youtube fab fa-youtube"></i>
                <i className="Linkdin fab fa-linkdin"></i>
                <i className="printerest fab fa-pinterest-p"></i> 
              </div>

              <div className="col-6">
                        <ul className="second-footer">
                            <li>About Us</li>
                            <li>Contact Us</li>
                            <li>Terms of use</li>
                            <li>Customers Support</li>
                            <li>Privacy Policy</li>
                        </ul>
                    </div>
            </div>
          );
        })}
      </section>
    </>
  );
};

export default Footer;
