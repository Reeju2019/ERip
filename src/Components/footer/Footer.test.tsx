import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import Footer from './Footer'

test('footer', () => {
  render(<Footer />)
})

describe('pnf rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<Footer />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<Footer />)
    ReactDOM.unmountComponentAtNode(div)
  })
})