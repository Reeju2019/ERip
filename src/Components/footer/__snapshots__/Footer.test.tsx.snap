// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pnf rendering specification SnapShot testing 1`] = `
<section
  className="footer"
>
  <div
    className="container-fluid g-0 footer-container"
    style={
      Object {
        "backgroundColor": "#24252a",
        "display": "flex",
        "paddingBottom": "50px",
        "paddingTop": "50px",
      }
    }
  >
    <div
      className="first-footer "
    >
      <img
        alt="white-logo"
        className="erip-logo"
        src="/assets/erip-logo-white.png"
      />
      <p
        className="footer-text"
      >
        Â© Erip 2021 - 2021, All Rights Reserved
      </p>
      <i
        className="fa-brands fa-twitter"
      />
      <i
        className="facebook fab fa-facebook-square"
      />
      <i
        className="Instagram fab fa-instagram"
      />
      <i
        className="youtube fab fa-youtube"
      />
      <i
        className="fa-brands fa-linkedin"
      />
      <i
        className="printerest fab fa-pinterest-p"
      />
    </div>
    <div
      className="col-6"
    >
      <ul
        className="second-footer"
      >
        <li>
          About Us
        </li>
        <li>
          Contact Us
        </li>
        <li>
          Terms of use
        </li>
        <li>
          Customers Support
        </li>
        <li>
          Privacy Policy
        </li>
      </ul>
    </div>
  </div>
</section>
`;
